.gitignore

```diff
@@ -10,8 +10,6 @@ output/*
 !output/.gitkeep
 # The config folder is intended to keep multiple program configs, these are sensitive to the system, a default is included for people to work off of while the others are hidden
 config/*
-!config/config.default.env
```

Comment:
These are now in the package so not needed here, user config files can be stored in this location still and should be based off the default